spec/mainadditions.html

```diff
@@ -271,6 +271,247 @@ <h1>AvailableNamedTimeZoneIdentifiers ( ): a List of Time Zone Identifier Record
         1. Return _result_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-systemtimezoneidentifier" oldids="sec-defaulttimezone" type="implementation-defined abstract operation">
+      <h1>SystemTimeZoneIdentifier ( ): a String</h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It returns a String representing the host environment's current time zone, which is either a String representing <del>a UTC offset for which IsTimeZoneOffsetString returns *true*</del><ins>an offset time zone identifier in its normal form</ins>, or a primary time zone identifier.
+        </dd>
+      </dl>
+
+      <emu-alg>
+        1. If the implementation only supports the UTC time zone, return *"UTC"*.
+        1. Let _systemTimeZoneString_ be the String representing the host environment's current time zone, either a primary time zone identifier or <del>an offset time zone identifier</del><ins>a |NormalizedUTCOffset|</ins>.
```

Comment:
```suggestion
        1. Let _systemTimeZoneString_ be the String representing the host environment's current time zone, either a primary time zone identifier or <del>an offset time zone identifier</del><ins>a String _s_ for which ParseText(StringToCodePoints(_s_), |NormalizedUTCOffset|) returns a Parse Node</ins>.
```

Reply:
I wrote this by copying a bunch of examples in 262 text using "is a |Foo|", which does seem clearer than using ParseText. Examples: 

* Each [[Site]] value is a Parse Node that is a |TemplateLiteral|
* String that when processed is a |FunctionBody|
* Assert: _lhs_ is a |LeftHandSideExpression|.
* Assert: _d_ is a |FunctionDeclaration|.

Is there a reason to use the longer text instead of the simpler one?

Reply:
Or is the difference that those 262 examples are Code Points and not Strings, so you need the extra wording?  If so, should we create a new AO e.g. `StringMatchesProduction` or something like that to make that case less wordy?

Reply:
After looking into this more, it's the String vs. Code Points thing. Suggestion accepted!

Reply:
See also https://github.com/tc39/ecma262/pull/3099 , which if accepted will allow for some minor improvement here.

Reply:
That 262 PR would simplify Temporal, good idea. In the meantime I'll just use your suggestion. Only problem: it causes a spec lint error: `could not find a preceding declaration for "s"`. Any idea what is alternate text that won't cause that error?

Reply:
Yep: https://tc39.es/ecmarkup/#variable-declarations
```
        1. <ins>[declared="s"]</ins> Let _systemTimeZoneString_ be the String representing the host environment's current time zone, either a primary time zone identifier or <del>an offset time zone identifier</del><ins>a String _s_ for which ParseText(StringToCodePoints(_s_), |NormalizedUTCOffset|) returns a Parse Node</ins>.
```

Reply:
None of those prevent the lint error. 😢 

How about I just leave it as "is a |NormalizedUTCOffset|" and hope that your 262 PR gets merged before the July plenary when we present this PR?

Reply:
Ok, but please also open an issue in https://github.com/tc39/ecmarkup. I suspect it is an \<ins>/\<del> problem.

Reply:
👍 https://github.com/tc39/ecmarkup/issues/528

Reply:
There's a workaround, which is to put the `[declared ...]` outside the `<ins>`. New text:

```
1. [declared="s"] Let _systemTimeZoneString_ be the String representing the host environment's current time zone, either a primary time zone identifier or an offset time zone identifier<ins> String _s_ for which ParseText(StringToCodePoints(_s_), |NormalizedUTCOffset|) returns a Parse Node</ins>.
```